code/model/Comment.php

```diff
@@ -691,6 +691,22 @@ public function AllReplies() {
 		return $list;
 	}
 
+    /**
+    * Determine if JavaScript is enabled, used in templates
+    * @return boolean true iff JavaScript enabled with include_js = true in config
+    */
+    public function getJavaScriptEnabled() {
+        return $this->getOption('include_js');
```

Comment:
I'm not sure "javascriptEnabled" is the right term here; Whether javascript is enabled or not is up to the client.

What the server actually decides is whether javascript-disabled devices are supported. ;) I would expect the default to be true, but it's fine to allow it to be false if the developer choses not to support these.